lti_consumer/tests/unit/plugin/test_views.py

```diff
@@ -34,11 +37,16 @@ def test_public_keyset_endpoint(self):
 
         # Check response
         self.assertEqual(response.status_code, 200)
-        # Check function call arguments
-        self._mock_xblock_handler.assert_called_once()
-        kwargs = self._mock_xblock_handler.call_args.kwargs
-        self.assertEqual(kwargs['usage_id'], self.location)
-        self.assertEqual(kwargs['handler'], 'public_keyset_endpoint')
+        self.assertEqual(response['Content-type'], 'application/json')
+        self.assertEqual(response['Content-Disposition'], 'attachment; filename=keyset.json')
+
+        # Check public keyset
+        self.lti_config.refresh_from_db()
+        retrieved_public_keyset = json.loads(response.content.decode('utf-8'))
+        self.assertEqual(
+            json.loads(self.lti_config.lti_1p3_platform_public_jwk),
+            retrieved_public_keyset
```

Comment:
Nit: Since you have to use json.loads on both values, it would be better to make them both inline calls here, just for readability.